test/controllers/users/emails_controller_test.rb

```diff
@@ -57,8 +92,30 @@ class Users::EmailsControllerTest < ActionController::TestCase
         assert_not_authorised
       end
 
+      should "authorize access if ApiUserPolicy#edit? returns true when user is an API user" do
+        user = create(:api_user)
+
+        api_user_policy = stub_everything("api-user-policy", edit?: true)
+        ApiUserPolicy.stubs(:new).returns(api_user_policy)
```

Comment:
It'd be good to consistently use `PunditHelpers#stub_policy` but I appreciate that it might be easier to make that change in one go outside of this PR.

Reply:
Sure thing - I don't think it existed when I started this branch!